testing/citest/tests/google_front50_test.py

```diff
@@ -64,7 +64,8 @@ def _do_init_bindings(self):
     context = ExecutionContext()
     config = self.agent.deployed_config
     enabled = config.get('spinnaker.gcs.enabled', False)
-    if not enabled:
+    # We don't read the config files while using 'native' platform.
+    if self.bindings['HOST_PLATFORM'] != 'native' and not enabled:
```

Comment:
I'm using `HOST_PLATFORM` for the k8s tests currently, I'd be real sad if you removed it. I can do your suggestion though.